invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/index.js

```diff
@@ -1,22 +1 @@
-import { compose } from 'redux';
-import { connect } from 'react-redux';
-import { withRouter } from 'react-router-dom';
-import DocumentMetadataComponent from './DocumentMetadata';
-import { fetchDocumentDetails } from '../../state/actions';
-
-const mapStateToProps = state => ({
-  documentDetails: state.documentDetails.data,
-  error: state.documentDetails.error,
-});
-
-const mapDispatchToProps = dispatch => ({
-  fetchDocumentDetails: itemPid => dispatch(fetchDocumentDetails(itemPid)),
-});
-
-export const DocumentMetadata = compose(
-  withRouter,
-  connect(
-    mapStateToProps,
-    mapDispatchToProps
-  )
-)(DocumentMetadataComponent);
+export { default as DocumentMetadata } from './DocumentMetadata';
```

Comment:
You have conflict on that file because deletion of Document was merged in the meanwhile. Please when you rebase keep your concept(don't take document data from store but only what is needed and remove the `compose`,`withRouter`).